scipy/signal/windows/_windows.py

```diff
@@ -336,6 +336,7 @@ def bohman(M, sym=True):
     >>> plt.figure()
     >>> A = fft(window, 2048) / (len(window)/2.0)
     >>> freq = np.linspace(-0.5, 0.5, len(A))
+    >>> np.seterr(divide='ignore')
```

Comment:
I would use the `with np.errstate(...)` context manager for these, it's nicer to contain the effect and make clear where/why it's needed

Reply:
There is/was #15333 about this  but I think we should make a better example indeed.

Reply:
There is/was #15333 about this  but I think we should make a better example indeed.

Reply:
@ilayn indeed did not see this issue! Do you have something in mind? Or shall I just change the `freq` linspace to `np.linspace(-0.45, 0.45, len(A))` or something like this?